sources/commands/Enable.ts

```diff
@@ -83,9 +83,9 @@ export class EnableCommand extends Command<Context> {
     const symlink = path.relative(installDirectory, path.join(distFolder, `${binName}.js`));
 
     if (fs.existsSync(file)) {
-      const currentSymlink = await fs.promises.realpath(file);
+      const currentSymlink = await fs.promises.readlink(file);
```

Comment:
The issue with that is that it's going to throw if `file` is not a symlink. We should probably add tests otherwise this kind of regression is bound to happen again in the future

Reply:
I've pushed a fix that uses lstat to check isSymbolicLink() before calling readlink, and it now correctly unlinks non-symlink files too.

I also added the tests.